src/v/kafka/server/handlers/sasl_authenticate.cc

```diff
@@ -21,7 +21,7 @@ ss::future<response_ptr> sasl_authenticate_handler::handle(
   request_context ctx, [[maybe_unused]] ss::smp_service_group g) {
     sasl_authenticate_request request;
     request.decode(ctx.reader(), ctx.header().version);
-    vlog(klog.debug, "Received SASL_AUTHENTICATE");
+    vlog(klog.debug, "Received SASL_AUTHENTICATE {}", request);
```

Comment:
squash into first commit

Reply:
I squashed it into the second commit; I'd prefer both commits to not include logged SASL data.